Makefile

```diff
@@ -1,5 +1,6 @@
 DEV_CORPUS := $(shell pwd)/corpus/20k-enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt
-REAL_CORPUS := "$(shell pwd)/corpus/enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt"
+#REAL_CORPUS := "$(shell pwd)/corpus/enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt"
+REAL_CORPUS := "/lucenedata/enwiki/enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt"
```

Comment:
I'm sure this served you well in your local setup. 

Not sure if we want to check this it in

Reply:
Woops, right, I didn't mean to -- I've now fixed my local area (used symlink) so I don't need that mod locally.